redis/src/test/scala/zio/redis/ApiSpec.scala

```diff
@@ -39,10 +44,170 @@ object ApiSpec extends BaseSpec {
+            newKey  <- uuid
+            renamed <- renameNx(key, newKey).either
+          } yield assert(renamed)(isLeft)
+        },
+        testM("unlink existing key") {
+          for {
+            key     <- uuid
+            value   <- uuid
+            _       <- set(key, value, None, None, None)
+            removed <- unlink(key, Nil)
+          } yield assert(removed)(equalTo(1L))
+        },
+        testM("touch two existing keys") {
+          for {
+            key1    <- uuid
+            value1  <- uuid
+            _       <- set(key1, value1, None, None, None)
+            key2    <- uuid
+            value2  <- uuid
+            _       <- set(key2, value2, None, None, None)
+            touched <- touch(key1, List(key2))
+          } yield assert(touched)(equalTo(2L))
+        },
+        testM("set followed by scan") {
+          for {
+            key             <- uuid
+            value           <- uuid
+            _               <- set(key, value, None, None, None)
+            scan            <- scan(0L, None, None, None)
+            (next, elements) = scan
+          } yield assert(next)(isNonEmptyString) && assert(elements)(isNonEmpty)
+        },
+        testM("fetch random key") {
+          for {
+            key       <- uuid
+            value     <- uuid
+            _         <- set(key, value, None, None, None)
+            allKeys   <- keys("*")
+            randomKey <- randomKey()
+          } yield assert(allKeys)(contains(randomKey.get))
+        },
+        testM("check value type by key") {
+          for {
+            key1       <- uuid
+            value1     <- uuid
+            _          <- set(key1, value1, None, None, None)
+            key2       <- uuid
+            value2     <- uuid
+            _          <- lPush(key2)(value2)
+            stringType <- typeOf(key1)
+            listType   <- typeOf(key2)
+          } yield assert(stringType)(equalTo("string")) && assert(listType)(equalTo("list"))
+        },
+        testM("dump followed by restore") {
+          for {
+            key      <- uuid
+            value    <- uuid
+            _        <- set(key, value, None, None, None)
+            dumped   <- dump(key)
+            _        <- del(key, Nil)
+            restore  <- restore((key, 0L, dumped, None, None, None, None)).either
+            restored <- get(key)
+          } yield assert(restore)(isRight) && assert(restored)(isSome(equalTo(value)))
         }
       )
     ).provideCustomLayerShared(Executor)
 
-  private val Executor = RedisExecutor.live("127.0.0.1", 6379).orDie
-  private val uuid     = UIO(UUID.randomUUID().toString)
+  private val Executor                                          = RedisExecutor.live("127.0.0.1", 6379).orDie
+  private val uuid                                              = UIO(UUID.randomUUID().toString)
+  private def durationInMillis(seconds: Long): UIO[Duration]    = UIO(Duration(seconds, TimeUnit.MILLISECONDS))
```

Comment:
You can remove the need for this by `import zio.duration._`. Every call to `durationInMillis` becomes `something.millis`.